css/stylesheet.css

```diff
@@ -0,0 +1,125 @@
+/*
+
+{font-family: Oswald}
+{font-family: Lato}
+{font-family: Merriweather}
+{font-family: Lora}
+*/
+
+.main-body {
+  margin-left: 12.5%;
+  margin-right: 12.5%;
+}
+
+.l-masthead {
+  margin-top:50px;;
+
+}
+.l-box-model {
+  width: 35%;
+  float: left;
+  margin-left: -10%;;
+
+}
+.l-feature {
+  width: 65%;
+  float: right;
+}
+.l-staying-afloat {
+  text-align:right;
+  width:15%;
+  padding:1%;
+  float:left;
+
+}
+.l-bullets {
+  width:70%;
+  padding:1%;
+  float:right;
+
+}
+.l-famous {
+  width: 100%;
+  display: inline-block;
+  text-align: center;
+
+
+  /* display: block; */
+}
+.l-bottom-line {
+  width:73%;
+  float:left;
+  margin-right:2%;
+
+}
+.l-transformers {
+  width:20%;
+  float:right;
+  padding-left:1.2%;
+  border-left:1px solid black;;
+}
+
+.l-overflow {
+  width:1060px;
+  margin-left:-12.5%;
+  margin-right:-12.5%;
+
+  /* overflow-y: visible; */;
+
+}
+.l-register {
+  width:960px;
+
+}
+.l-footer {
+  width:960px;
+
+}
+
+
+
+/*=====FONTS=====*/
```

Comment:
I see what you did here with the OOCSS/SMACSS approach to type, but I would also suggest assigning a default font type to the `html` element so that there is a default type without having to always assign.